Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -19,7 +19,7 @@ This will result in the following:
 # Install latest CLI tools
 
 * Download latest CLI tools from [https://github.com/dotnet/cli/](https://github.com/dotnet/cli/) and add them to the path. The latest CLI tools include MSBuild support that the native compilation build integration depends on. These instructions have been tested with build `1.0.0-rc4-004812`.
-* On windows ensure you are using the 'VS2015 x64 Native Tools Command Prompt'
+* On windows ensure you are using the 'x64 Native Tools Command Prompt for VS 2017' or 'VS2015 x64 Native Tools Command Prompt'
```

Comment:
There are a couple more references to 2015 in this file. Should we update them too and add 2017?

Or should we just drop all references to 2015? We still need to support it because of the CI, but does the reproNative project still build in 2015 after your changes?